contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -491,6 +491,17 @@ extern int
 static void
 pltsql_exec_function_cleanup(PLtsql_execstate *estate, PLtsql_function *func, ErrorContextCallback *plerrcontext);
 
+bool		called_for_tsql_itvf_function = false;
+bool  		called_for_tsql_itvf_func(void);
+
+
+bool called_for_tsql_itvf_func()
```

Comment:
Nit: datatype and function name should be in separate lines.

Reply:
done